GiwazipClient/Views/SplashViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SplashViewController.swift
+//  GiwazipClient
+//
+//  Created by 김민택 on 2023/01/17.
+//
+
+import UIKit
+
+import SnapKit
+
+class SplashViewController: BaseViewController {
+
+    // MARK: - Property
+
+    private let enterButton: UIButton = {
+        $0.setTitle("입장하기", for: .normal)
+        $0.setTitleColor(.black, for: .normal)
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .semibold)
+        $0.backgroundColor = .gray
+        $0.layer.cornerRadius = 16
+
+        return $0
+    }(UIButton())
+
+    // MARK: - Method
+
+    override func layout() {
+        super.layout()
+
+        view.addSubview(enterButton)
+        enterButton.snp.makeConstraints { make in
+            make.left.equalTo(view.safeAreaLayoutGuide).offset(16)
+            make.right.bottom.equalTo(view.safeAreaLayoutGuide).offset(-16)
+            make.height.equalTo(50)
+        }
```

Comment:
p3;
1. 혹시 $0 이런 방법으로 layout 잡는 건 어떠할까요??! 저는 이게 더 편하긴 합니다. 내일 함께 이야기해보아요!
2. offset 대신 inset을 이용하면 한 번에 처리할 수 있습니다!

```suggestion
        enterButton.snp.makeConstraints {
            $0.left.bottom.right.equalTo(view.safeAreaLayoutGuide).inset(16)
            $0.height.equalTo(50)
        }
```

Reply:
> p3;
> 
> 1. 혹시 make 대신 $0 이런 방법으로 layout 잡는 건 어떠할까요??! 저는 이게 더 편하긴 합니다. 내일 함께 이야기해보아요!
> 2. offset 대신 inset을 이용하면 한 번에 처리할 수 있습니다!

오 저도 이런 식으로 할 수는 없나 고민하고 있었는데, 방법을 못찾고 있었는데 디너가 제시한 방법이 훨씬 깔끔하고 좋은 것 같네요. 굳~